doc/en/user/source/configuration/status.rst

```diff
@@ -100,3 +100,203 @@ Module Status Field Descriptions
    Module Status popup
      
 
+System Status
+-------------
+
+System Status adds some extra information about the system in the GeoServer status page in a tab named ``Monitoring``
```

Comment:
That tab shoudl be renamed ``System Status``, in the doc and in the code.

Reply:
Renamed.